documentation/cloud-manager-guide/secure-data-connectors/manage-data-connectors.md

```diff
@@ -0,0 +1,66 @@
+---
+keywords:
+title: Manage Data Connectors
+description: Manage data connectors
+---
+# Manage Data Connectors
+
+Secure data connectors can be managed from the *Secure Data Connectors* screen. From here, you can edit, enable or disable, and delete connectors. This guide outlines the steps to manage data connectors.
```

Comment:
Can we just say "Data connectors can be managed..." or do we need to say "secure data connectors" twice?